tests_integ/helpers/deployer/utils/time.py

```diff
@@ -0,0 +1,127 @@
+"""
+Date & Time related utilities
+"""
+
+import datetime
+import dateparser
+
+from dateutil.tz import tzutc
+
+
+def timestamp_to_iso(timestamp):
```

Comment:
Should mention all timestamps here are milliseconds (why?); docstring suggests second-precision. Python timestamps are usually second-precision floats.